client/landing/stepper/declarative-flow/internals/steps-repository/site-creation-step/index.tsx

```diff
@@ -147,7 +145,10 @@ const SiteCreationStep: Step = function SiteCreationStep( { navigation, flow, da
 			theme,
 			siteVisibility,
 			urlData?.meta.title ?? selectedSiteTitle,
-			siteAccentColor,
+			// We removed the color option during newsletter onboarding.
+			// But backend still expects/needs a value, so supplying the default.
+			// Ideally should remove this and update code downstream to handle this.
+			'#113AF5',
```

Comment:
Let's follow up with backend to not send this and clean up afterwards. AFAIK other flows aren't using this.  👍